package.json

```diff
@@ -53,7 +53,7 @@
     "test:integration": "vitest run tests/integration/",
     "test:unit": "vitest run tests/unit/",
     "postinstall": "node ./scripts/postinstall.js",
-    "prepublishOnly": "npm shrinkwrap",
+    "prepublishOnly": "npm install --omit=dev && npm shrinkwrap",
```

Comment:
hrm, turns out this won't work... https://github.com/npm/cli/issues/4323. We should maybe just bring in https://www.npmjs.com/package/clean-publish 😢.

Reply:
Don't we need to just add an `npm prune --production` step to the release workflow?